scripts/start_argilla_server.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+set -e
+
+# Check elasticsearch response status
+status_code=$(curl -s -o /dev/null -w "%{http_code}" "$ARGILLA_ELASTICSEARCH")
+
+if [[ "$status_code" -ne 200 ]]; then
+  echo "ElasticSearch connection error. Returned status code: $status_code"
+  exit 1
+else
+  echo "ElasticSearch connected"
+fi
+
+# Run argilla-server
+uvicorn argilla:app --host "0.0.0.0" --port 6900
```

Comment:
Since we're fixing the port here, could you also add the `EXPOSE` statement in dockerfile?

Also, we need to update the provided [docker-compose.yml](https://github.com/argilla-io/argilla/blob/ci/optimize-docker-images/docker-compose.yaml#L8) and update the mapped ports for argilla. Please, review it and do the proper changes.

Reply:
Made the changes